From 447298791b6e557484e33d3f7c0bc085a3093dc0 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 29 Nov 2012 15:53:15 +0100
Subject: [PATCH] systemd/dracut-cmdline.service: run before
 systemd-vconsole-setup

cmdline does generate vconsole.conf, which is needed by vconsole-setup
---
 modules.d/98systemd/dracut-cmdline.service | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules.d/98systemd/dracut-cmdline.service b/modules.d/98systemd/dracut-cmdline.service
index 75f9b7e..4aba918 100644
--- a/modules.d/98systemd/dracut-cmdline.service
+++ b/modules.d/98systemd/dracut-cmdline.service
@@ -12,6 +12,7 @@ Description=dracut cmdline hook
 Documentation=man:dracut-cmdline.service(8)
 DefaultDependencies=no
 Before=dracut-pre-udev.service
+Before=systemd-vconsole-setup.service
 After=systemd-journald.socket
 Wants=systemd-journald.socket
 ConditionPathExists=/etc/initrd-release
